dsew_community_profile/tests/test_pull.py

```diff
@@ -32,151 +39,208 @@ def _assert_frame_equal(df1, df2, index_cols: List[str] = None):
-                    False),
-            example("NAAT positivity rate - last 7 days - ages <5",
-                    False),
-            example("Total RT-PCR diagnostic tests - last 7 days (may be an underestimate due to delayed reporting)",
-                    True),
-            example("Viral (RT-PCR) lab test positivity rate - last 7 days (may be an underestimate due to delayed reporting)",
-                    True),
-            example("RT-PCR tests per 100k - last 7 days (may be an underestimate due to delayed reporting)",
-                    False),
-            example("Confirmed COVID-19 admissions - last 7 days",
-                    True),
-            example("Confirmed COVID-19 admissions - percent change",
-                    False),
-            example("Confirmed COVID-19 admissions - last 7 days - ages <18",
-                    False),
-            example("Confirmed COVID-19 admissions - last 7 days - age unknown",
-                    False),
-            example("Confirmed COVID-19 admissions per 100 inpatient beds - last 7 days",
-                    False)
+            example(
+                "Total NAATs - last 7 days (may be an underestimate due to delayed reporting)", True
+            ),
+            example(
+                "Total NAATs - previous 7 days (may be an underestimate due to delayed reporting)",
+                True,
+            ),
+            example(
+                "NAAT positivity rate - last 7 days (may be an underestimate due to delayed reporting)",
+                True,
+            ),
+            example(
+                "NAAT positivity rate - previous 7 days (may be an underestimate due to delayed reporting)",
+                True,
+            ),
+            example(
+                "NAAT positivity rate - absolute change (may be an underestimate due to delayed reporting)",
+                False,
+            ),
+            example("NAAT positivity rate - last 7 days - ages <5", False),
+            example(
+                "Total RT-PCR diagnostic tests - last 7 days (may be an underestimate due to delayed reporting)",
+                True,
+            ),
+            example(
+                "Viral (RT-PCR) lab test positivity rate - last 7 days (may be an underestimate due to delayed reporting)",
+                True,
+            ),
+            example(
+                "RT-PCR tests per 100k - last 7 days (may be an underestimate due to delayed reporting)",
+                False,
+            ),
+            example("Confirmed COVID-19 admissions - last 7 days", True),
+            example("Confirmed COVID-19 admissions - percent change", False),
+            example("Confirmed COVID-19 admissions - last 7 days - ages <18", False),
+            example("Confirmed COVID-19 admissions - last 7 days - age unknown", False),
+            example("Confirmed COVID-19 admissions per 100 inpatient beds - last 7 days", False),
```

Comment:
yea, the main thing it's trying to do here is appease the line-length constraints. since to lines vary in length by a lot, there wont be a single setting that will make them all consistent (except for making them all one line, which is also something we don't want)